server/controllers/recipes.js

```diff
@@ -76,10 +90,15 @@ export default {
         pageCount: Math.ceil(recipeList.count / limitValue),
         pageSize: parseInt(recipeList.rows.length, 10),
```

Comment:
Just a preference, I think it will be nice to have the pagination metadata encapsulated in an object like:
```
pagination: {
    totalCount: recipeList.count,
    pageCount: Math.ceil(recipeList.count / limitValue),
    pageSize: parseInt(recipeList.rows.length, 10)
}
```
And also if you can do your logic outside the response object sent back, I think it will be cleaner and better.